### DISCUSSION TITLE
Создать демо-пример с использованием pgBouncer

### DISCUSSION THREAD

**Discussion by mao29** [Flexberry/rfcs#26] [Ideas]
Предлагаю сделать демо-пример для приложения, созданного с помощью Flexberry Platform, использующего pgBouncer. При этом необходимо настроить pgBouncer таким образом, чтобы для разных БД (прикладная, полномочия, аудит) использовались различные учетные записи и в то же время при подключении к самому pgBouncer-у тоже использовалась не одна учетная запись с правами на все. 

Добавление в дефолтный стек сервисов корректно настроенного pgBouncer-а при генерации позволит не менять существенно настройки PostgreSQL по умолчанию для увеличения максимального количества подключений.

**Comment by Anisimova2020** [comment]
https://github.com/Flexberry/Flexberry.PgBouncer.Sample
